src/protocol/modulus_conversion/convert_shares.rs

```diff
@@ -106,11 +106,44 @@ impl ConvertShares {
             });
         try_join_all(futures).await
     }
+
+    #[allow(dead_code)]
```

Comment:
This doesn't get called from the other code.  That seems like a real problem.

Reply:
Not sure if I follow. For now, we have left the execute function here in case it would be needed. We plan to purge it if finally it is not needed.